### PR TITLE
Add documentation on dead time

### PR DESCRIPTION
**Pull request by clly**
There wasn't any documentation for this anywhere in the README I think.  I've added it in this commit.  It includes example values and valid units from the godocs. I didn't make an issue since it was just docs and pretty simple but let me know if you want me to and I can take care of it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,13 @@ the only thing on the line:
 
           # A dictionary of fields to annotate on each event.
           "fields": { "type": "syslog" }
+
+          # how long to keep following a file if it's unchanged
+          # It will refollow the file if it changes but it closes 
+          # file handle. 
+          # example values: "300ms", "-1.5h" or "2h45m"
+          # valid units: "ns", "us" (or "µs"), "ms", "s", "m", "h".
```
Comment (jordansissel) [inline README.md]: I like the idea here, though I think subsecond units should be removed given it is probably not appropriate to have a file with a dead time of 4ns.

I also propose a revised documentation to make it read more clearly (at least to me)

> To save on resources (open files), logstash-forwarder will close file handles
> on files that have not emitted new events after some time period.
> This setting controls that time period which we call 'dead time'.
> The file will still be monitored for new events, but just not as an
> open file.
>
> Valid units include "s" (seconds), "m" (minutes), and "h" (hours). 
> For example, "30m" is 30 minutes.

Thoughts?

**Comment by clly** [comment]
Sounds better.  I think we should make sure people know they can do partials like 1h30m

**Comment by jordansissel** [comment]
@clly +1 to putting a compound example like that in the doc.

File: README.md [hunk 2]
```diff
@@ -81,6 +81,15 @@ the only thing on the line:
...
           "fields": { "type": "syslog" }
+
+          # To save on resources (open files), logstash-forwarder will close file handles
+          # on files that have not emitted new events after some time period.
+          # This setting controls that time period which we call 'dead time'.
+          # The file will still be monitored for new events, but just not as an
+          # open file.
+
+          # Valid units include "s" (seconds), "m" (minutes), and "h" (hours). 
+          # For example, "30m" is 30 minutes and "1h30m" is 1 hour and 30 minutes
```
Comment (jordansissel) [inline README.md]: I liked your example entry, can we put it back?

```
"dead time": "1h"
```

**Comment by jordansissel** [comment]
Other than my inline comment, I love the proposal here. +1

I'll merge it once the comment is resolved. Thank you for putting in the energy for improving user experience! :)

**Comment by clly** [comment]
Can't believe I accidentally pulled that out.  I've readded it

**Comment by jordansissel** [comment]
@clly  I forgot to ask before, can you do step 2 of the contributing guide? https://github.com/elasticsearch/logstash-forwarder/blob/master/CONTRIBUTING.md#contribution-steps

**Comment by clly** [comment]
I've signed the contributing guide

**Comment by frutik** [comment]
would be great to have explanation that if you need more than 24h you can use 48h, 120h and so. I was not obvious at least for me.

**Comment by babadofar** [comment]
+1

**Comment by jordansissel** [comment]
@clly Seems you signed the CLA as an email address that doesn't match the  author on your commits - you can see the commit author here in the patch: https://github.com/elastic/logstash-forwarder/pull/401.patch - can you fix that?

**Comment by clly** [comment]
Ahhhhhh, that makes sense.  Didn't even realize I forgot to reset that.  Easily fixable.  I did a force push with the right author

**Comment by ruflin** [comment]
@clly Thank you for your contribution. This should be improved in filebeat, the next version of lsf. dead time is now called ignore_older. Here are the docs: https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-configuration-details.html#_ignore_older I'm closing this PR now.

@dedemorton I like the part about the open files and why ignore_older is needed. Should we add this too?

**Pull request closed without merging**